apps/sudoku-frontend/src/app/app-routing.module.ts

```diff
@@ -29,6 +29,10 @@ const routes: Routes = [
     loadChildren: () => import('./pages/login/login.module').then(m => m.LoginModule),
     canActivate: [AuthGuard]
   },
+  {
+    path: 'classic-game',
+    loadChildren: () => import('./pages/classic-game/classic-game.module').then(m => m.ClassicGameModule)
```

Comment:
Den AuthGuard benutzen so wie mein Menu mit
`canActivate: [AuthGuard]`

Sonst kann man den login bypassen indem man die route einfach in die Url eingibt

Reply:
done